packages/cfd/src/Components/Routes/binary-routes.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import getRoutesConfig from '../../Constants/routes-config';
+import RouteWithSubRoutes from './route-with-sub-routes.jsx';
+
+const BinaryRoutes = props => {
+    const { is_dashboard } = React.useContext(PlatformContext);
+
+    return (
+        <React.Suspense
+            fallback={() => {
+                return (
+                    <div>
+                        <Localize i18n_default_text='Loading...' />
+                    </div>
+                );
+            }}
+        >
+            <Switch>
+                {getRoutesConfig({ is_dashboard }).map((route, idx) => (
+                    <RouteWithSubRoutes key={idx} {...route} {...props} />
```

Comment:
```suggestion
                    <RouteWithSubRoutes key={idx} {...route} {...props} />
```
The `key` should not use idx generated by `map()` can we use something unique from the return of `getRoutesConfig`